doc/sphinx/source/community/detailed_release_procedure.rst

```diff
@@ -0,0 +1,190 @@
+.. _detailed-release-procedure:
+
+Release: recipes runs and comparison
+====================================
+
+The release procedure for ESMValTool is a fairly involved process (at the moment), so it
+is important to be very well organized and to have documented each procedural steps, so that
+the next release manager can follow said steps, and finalize the release without any delays.
+
+The workflow below assumes an ESMValCore release candidate, or a completed stable release, have been released
+and deployed on conda-forge and PyPI; it also assumes the release manager has access to accounts on DKRZ/Levante.
+
+Below is a list of steps that the release manager, together with the previous release manager, should go through before the actual release;
+these include testing the new code by running all available recipes in yhe `main` branch, and comparing the output against
+the previous release.
+
+Open an issue on GitHub
+-----------------------
+
+First, open an issue on GitHub where the testing workflow before the release is documented (see example https://github.com/ESMValGroup/ESMValTool/issues/2881).
```

Comment:
A discussion might be more suitable for this purpose, as it allows organized conversation about multiple different topics. E.g. for every broken recipe, a new thread could be started.

Reply:
I'm not so convinced about the usage of discussions. If we do several rounds of recipe testing (likely to be the case for v2.8), it might be simpler to open/close issues for each round. It might not be clear for the recipe maintainers which discussion is the active one and threads would spread across several discussions.

Reply:
I too agree with Remi (and implicitly, with myself haha) - Issues are: 1. more visible and 2. closable and forgettable and searchable, plus - the "reply to a reply" format in Discussions may lead to bits that are missed

Reply:
It;s up to you, but I personally find an issue becomes very difficult to read if you're trying to discuss several issues at the same time in it. If this happens at a high pace (several messages a day), it becomes quite impossible to follow for relative outsiders such as recipe maintainers or other core team members not as actively involved in the release. If you really prefer issues, my recommendation would be to open an issue for each broken recipe separately and try to limit the conversation in the overview issue somehow.